### ISSUE TITLE
Add teammates to solve announcement

### ISSUE THREAD

**Issue by Corb3nik** [OpenToAllCTF/OTA-Challenge-Bot#7]
When announcing a solved challenge through the `solve` command, a member should be able to mention which teammates helped the solve.

Something like this :
```
@ota_bot solve "web 300" @grazfather
ota_bot : *web 300* : @corb3nik solved "web 300" with the help of @grazfather.
```

**Comment by Corb3nik** [comment]
Tried the following command : 
```
!solved web200 kileak
```
... in a challenge channel and in a CTF channel. I get the following error message : 
```
2017-08-27 14:42:36,533 - botserver            - Received bot command : solved web200 kileak (C6TM2CDR6)
2017-08-27 14:42:36,533 - handler_factory      - Processing message: solved web200 kileak from C6TM2CDR6 (U1EUDSN5S)
2017-08-27 14:42:36,533 - handler_factory      - An error has occured while processing a command: name 'arggs' is not defined
```

**Comment by Kileak** [comment]
fixed by #23
